code/src/Components/ThoughtsForm.js

```diff
@@ -0,0 +1,43 @@
+import React, {useState} from 'react'
+
+import './ThoughtsForm.css'
+
+export const ThoughtsForm = ({onMessageChange}) => {
+    const [newMessages, setNewMessages] = useState("");
+
+    /*onHandleSubmit handling what happens when the form is submitted. 
+    Sending the inputted data to the postSingleMessage function is App.js via the onMessageChange prop. 
+    
+    Also clearing what the user has inputted in the textarea by setting hte newMessages back to an empty string by way of the setNewMessages function once the form is submitted. This is supported in the textarea value attribute where it's set to the newMessage state*/
+
+    const handleSubmit = event => {
+        event.preventDefault();
+        onMessageChange(newMessages);
+        setNewMessages("");
+    };
+
+    return (
+        <article className="message-container form-container">
+            <h3>What's making you happy right now?</h3>
+            <form onSubmit={handleSubmit}>
+                <textarea
+                className="text-box"
+                id="happyThought"
```

Comment:
this maybe correct, but of curiosity what do you have to "id" for?

Reply:
@nathalienordstrom I used id instead of class name as the css wouldn't implement on the text-area when using the class name for some strange reason. Worked around it by using id and it worked 😊